nginx.conf

```diff
@@ -0,0 +1,81 @@
+events {
+    worker_connections 1024;
+}
+
+# frontend
+http {
+    server {
+        listen 80;
+        server_name sim.genlayer.com;
```

Comment:
@YeagerAI-Bulat we probably need to change this to the new subdomain

Reply:
Retaining the current simulator URL for compatibility as we plan to deploy additional simulators in the future.
Refer to this Jinja template: https://github.com/yeagerai/genlayer-simulator-infra/blob/f7fb254492d52bb692e55dca886b7a9525579337/roles/genlayer/templates/nginx.j2#L9

Reply:
Retaining the current simulator URL for compatibility as we plan to deploy additional simulators in the future.
Refer to this Jinja template: https://github.com/yeagerai/genlayer-simulator-infra/blob/f7fb254492d52bb692e55dca886b7a9525579337/roles/genlayer/templates/nginx.j2#L9